src/sentry/rules/conditions/first_seen_event.py

```diff
@@ -13,3 +18,16 @@ def passes(self, event: Event, state: EventState) -> bool:
             return state.is_new
         else:
             return state.is_new_group_environment
+
+    def get_activity(self, start: datetime, end: datetime) -> Sequence[Activity]:
+        return [
+            Activity(
+                project=g.project,
+                group=g,
+                type=ActivityType.FIRST_SEEN.value,
+                datetime=g.first_seen,
+            )
```

Comment:
It's a bit weird to populate `Activity` with data from group. Should we design a separate dataclass that we use to get activity related to a condition?

Reply:
That sounds like a better choice